services/metadata_service/tests/integration_tests/utils.py

```diff
@@ -209,17 +220,32 @@ async def assert_api_get_response(cli, path: str, status: int = 200, data: objec
         http status code to expect from response
     data : object
         An object to assert the api response against.
+    data_is_unordered_list : bool
+        Data is an unordered list, so ignore ordering when comparing data and response body
     """
     response = await cli.get(path)
 
     assert response.status == status
 
-    if data:
-        body = json.loads(await response.text())
+    body = json.loads(await response.text())
+    if data is None:
+        return
+    if data_is_unordered_list:
+        assert type(data) == list and type(body) == list
+        if not data:
+            assert body == []
+            return
```

Comment:
does it need a special case here? i thought the assertion a few lines below would still handle that?

Reply:
You are right, will remove that.  Some previous version was looking for the first element to detect fields (not needed after all).